src/content/peerconnection/trickle-ice/index.html

```diff
@@ -82,12 +82,13 @@
       <h2>ICE options</h2>
 
       <div id="iceTransports">
-        <span>IceTransports value:</span>
-        <input type="radio" name="transports" value="all" id="all" checked><label for="all">all</label>
+        <label for="ipv6"><span>IceTransports value:</span></label>
+        <input type="radio" name="transports" value="all" id="all" checked>
+        <span>all</span>
         <input type="radio" name="transports" value="relay" id="relay">
-        <label for="relay">relay</label>
+        <span>relay</span>
         <input type="radio" name="transports" value="none" id="none">
-        <label for="none">none</label>
+        <span>none</span>
```

Comment:
none has actually been removed from the spec, can you remove it as a drive-by? :-)